tern/classes/docker_image.py

```diff
@@ -146,11 +150,13 @@ def load_image(self):
             layer_count = 1
             while layer_diffs and layer_paths:
                 layer = ImageLayer(layer_diffs.pop(0), layer_paths.pop(0))
-                layer.set_checksum(checksum_type, layer.diff_id)
-                layer.gen_fs_hash()
-                layer.layer_index = layer_count
+                if super().load_until_layer >= layer_count or super().load_until_layer == 0:
```

Comment:
Same question about super here as I had before.. we can use `self._load_until_layer >= layer_count or self._load_until_layer == 0` just to be consistent. Could you also add a comment here that describes what you're doing with this check? Something like 
```
# Only load metadata for the layers we need to report on according to the --layers command line option
# If  --layers option is not present, load all the layers
```